web/models/account.ex

```diff
@@ -10,9 +10,10 @@ defmodule Entice.Web.Account do
   end
 
 
-  def changeset(account, params \\ :empty) do
+  def changeset(account, params \\ :invalid) do
```

Comment:
They did use %{} in their example but they also said it's either in the explanation above and since they had said :empty was replaced :invalid earlier I had already done it.